### DISCUSSION TITLE
SSL Error ERR_SSL_PROTOCOL_ERROR

### DISCUSSION THREAD

**Discussion by mkd0r** [vercel/vercel#10917] [Help]
### Question

Hello
I have an application installed on the https://garajpro-com-app-kadirmoglu.vercel.app/ domain. I am getting ERR_SSL_PROTOCOL_ERROR error in this application. How can I solve this? . The problem was not resolved when I deployed again or started a new project

**Comment by qublaidev** [comment]
Can you try accessing the site using a VPN I think there is a problem accessing it from Türkiye I have the same problem currently, but I can access it using VPN

**Comment by serter1** [comment]
I have that same problem.

**Comment by amyegan** [comment]
Thanks for reporting this. This error is related to an ISP level block.  Please follow in the main thread for the latest updates: https://github.com/vercel/community/discussions/5102

**Comment by rathod-priyanka** [comment]
The SSL Error ERR_SSL_PROTOCOL_ERROR is a common error message that occurs when there is an issue with the SSL/TLS (Secure Sockets Layer/Transport Layer Security) protocol during a secure connection attempt. This error typically occurs in web browsers, such as Google Chrome, Mozilla Firefox, or Microsoft Edge, when a user tries to access a secure website.

I recommend you please read this blog once, you have to get the proper answer https://www.ssl2buy.com/wiki/how-to-fix-err-ssl-protocol-error
